src/components/product/migrations/SplitScriptureSpanningBooks.migration.ts

```diff
@@ -130,7 +130,11 @@ export class SplitScriptureSpanningBooksMigration extends BaseMigration {
       // refs: labelOfScriptureRanges(product.scriptureReferences),
       product,
     });
-    return ix(this.splitRefsByBook(product.scriptureReferences))
+    return ix(
+      this.splitRefsByBook(
+        product.scriptureReferences as unknown as DbScriptureReferences,
+      ),
+    )
```

Comment:
I want to call this typing out specifically here @CarsonF - It feels like there is something downstream like the others we removed, but I couldn't find it. This doesn't feel correct, but outside of this, I wasn't sure where to go with a migration.

Reply:
We should probably just delete all our existing migrations at this point. They've all been ran where needed and we shouldn't need to maintain them anymore.

Reply:
Neo4j ignorance here, but guessing we don't need these migrations for new dev setup? I know in the SQL world you have to keep migrations forever. Is that not the case in the graph world?

Reply:
Both SQL/EdgeDB have schemas. So the migrations _mainly_ handled changes to those. Neo4j doesn't have a schema (on write), so these migrations only handled moving data around to the new format. As new DB's don't have this old data format to pull forward, the migrations are just ignored.